### PR TITLE
convert the literature review website into a lab website

### PR DESCRIPTION
**Pull request by jodoin**

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -11,8 +11,8 @@ hidden:     true  # Excluded from the menu
 </style>
 
 
-**Welcome!**  on the literature review archive of the [Videos & Images Theory and Analytics Laboratory (VITAL) of Sherbrooke University](http://vital.dinf.usherbrooke.ca/), headed by Professor [Pierre-Marc Jodoin](http://info.usherbrooke.ca/pmjodoin).  Our lab focuses on image processing, computer vision, video analytics, medical imaging, and statistical modeling. If you are interested in collaborating with us, please do not hesitate to [contact us](http://vital.dinf.usherbrooke.ca/contact/)!
+**Welcome!**  on the Videos & Images Theory and Analytics Laboratory (VITAL) website of Sherbrooke University, headed by Professor [Pierre-Marc Jodoin](http://info.usherbrooke.ca/pmjodoin).  Our lab focuses on image processing, computer vision, video analytics, medical imaging, and statistical modeling. If you are interested in collaborating with us, please do not hesitate to [contact us](http://info.usherbrooke.ca/pmjodoin)!
```
Comment (ThierryJudge) [inline index.md:14]: Maybe change this to `welcome **to** the`... ?

File: _config.yml [hunk 2]
```diff
@@ -13,10 +13,10 @@
 # you will see them accessed via {{ site.title }}, {{ site.email }}, and so on.
 # You can create any custom variable you would like, and they will be accessible
 # in the templates via {{ site.myvariable }}.
-title: VITAL Literature Review
+title: VITALab site
```
Comment (ThierryJudge) [inline _config.yml:16]: The title could be just `VITALab`

**Review by ThierryJudge — COMMENTED** [review]
Looks good!

File: _includes/footer.html [hunk 3]
```diff
@@ -12,10 +12,7 @@ <h2 class="footer-heading">{{ site.title | escape }}</h2>
       <div class="footer-col footer-col-2">
         <ul class="social-media-list" style="line-height: 1.8rem">
           <li>
-            <a href="https://bitbucket.org/vitalab/vitalabai_public">Bitbucket</a>
-          </li>
-          <li>
-            <a href="http://vital.dinf.usherbrooke.ca/">Website</a>
+            <a href="https://github.com/vitalab/vitalab.github.io">gitHub</a>
```
Comment (nathanpainchaud) [inline _includes/footer.html:15]: Wrong capitalization:
```suggestion
            <a href="https://github.com/vitalab/vitalab.github.io">GitHub</a>
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,4 +1,4 @@
-# Contributing to the VITAL Literature Review
+# Contributing to the VITALab website
 
 In order to contribute to with your reviews to the VITAL Literature Review,
```
Comment (nathanpainchaud) [inline CONTRIBUTING.md:3]: There's a typo here with the "contribute to with", and I'd shorten it to:
```suggestion
In order to contribute reviews to the VITAL Literature Review,
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -1,4 +1,4 @@
-# Contributing to the VITAL Literature Review
+# Contributing to the VITALab website
```
Comment (nathanpainchaud) [inline CONTRIBUTING.md:2]: Maybe add a lower level header here? I understand that the file should remain general for the whole website, but right now we don't support contributions to anything but the literature reviews, and that's all the instructions linked to cover. So maybe something like:
```suggestion
# Contributing to the VITALab website

## Literature Reviews
```

**Review by nathanpainchaud — COMMENTED** [review]
Some typos + suggestion to structure the contributing guidelines.

**Review by nathanpainchaud — APPROVED** [review]

**Review by ThierryJudge — APPROVED** [review]

**Comment by AntoineTheb** [comment]
Maybe add the list of members ? Not sure what the plans are for the existing website: http://vital.dinf.usherbrooke.ca/members/

**Pull request merged**